examples/submodule_vpc_serverless_connector/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version = "~> 3.62"
+}
+
+provider "google-beta" {
+  version = "~> 3.62"
+}
+
+module "serverless-connector" {
```

Comment:
It could be better to showcase how this module works with the main module similar to other submodule examples like [peering](https://github.com/terraform-google-modules/terraform-google-network/blob/master/examples/submodule_network_peering/main.tf), [fw](https://github.com/terraform-google-modules/terraform-google-network/blob/master/examples/submodule_firewall/main.tf) etc

Reply:
Added module for simple vpc deployment